basis-universal-sys/build.rs

```diff
@@ -15,10 +15,12 @@ fn build_with_common_settings() -> cc::Build {
 }
 
 fn main() {
+    let sse_support = cfg!(target_feature = "sse4.1");
```

Comment:
Note that `cfg!` will detect the features used to build the build script itself, which are not necessarily the same as the target building for (for example when `--target` is used Cargo will not pass the rustflags to the build script, but it will pass them to the crate you're building for). It's suggested instead to read the `CARGO_CFG_TARGET_FEATURE` environment variable, which will properly report the target features enabled for the target you're building for.

Something like:

```rust
let sse4.1 = std::env::var("CARGO_CFG_TARGET_FEATURE")
	.expect("Cargo didn't set the CARGO_CFG_TARGET_FEATURE env var")
	.split(',')
	.any(|feature| feature == "sse4.1");

```